File: AspNet4/Mvc/AdvancedSearch/Controllers/EasyQuery/AdvancedSearchController.cs
using System;
using System.Configuration;
using System.Web.Http;
using System.Data.SqlClient;

using Korzh.EasyQuery.Services;
using Korzh.EasyQuery.AspNet;

using EqDemo.Models;

namespace EqDemo.Controllers
{
    [RoutePrefix("api/easyquery")]
    public class AdvancedSearchController : EasyQueryApiController
    {
        protected override void ConfigureEasyQueryOptions(EasyQueryOptions options)
        {
            options.UseManager<EasyQueryManagerSql>();
            options.DefaultModelId = "nwind";
            options.BuildQueryOnSync = true;
            options.SaveQueryOnSync = false;

            options.UseDbContext(ApplicationDbContext.Create());

            // Uncomment this line if you want to load model directly from connection 
            // Do not forget to uncomment SqlClientGate registration in WebApiConfig.cs file
            //options.UseDbConnectionModelLoader();

            var path = System.Web.Hosting.HostingEnvironment.MapPath("~/App_Data");
            options.UseQueryStore((_) => new FileQueryStore(path));
        }
    }
}

File: AspNet4/WebForms/VB.AdvancedSearch/Controllers/AdvancedSearchController.vb
Imports System
Imports System.Configuration
Imports System.Data.SqlClient
Imports System.Web.Http

Imports Korzh.EasyQuery.AspNet
Imports Korzh.EasyQuery.Services

<RoutePrefix("api/easyquery")>
Public Class AdvancedSearchController : Inherits EasyQueryApiController

    Protected Overrides Sub ConfigureEasyQueryOptions(options As EasyQueryOptions)
        options.UseManager(Of EasyQueryManagerSql)()

        options.DefaultModelId = "nwind"
        options.BuildQueryOnSync = True

        options.UseDbContext(ApplicationDbContext.Create())
        ' Do Not forget to uncomment SqlClientGate registration in WebApiConfig.cs file
        ' options.UseDbConnectionModelLoader();

        Dim path = System.Web.Hosting.HostingEnvironment.MapPath("~/App_Data")
        options.UseQueryStore(Function(__) New FileQueryStore(path))
    End Sub
End Class
